arcade/experimental/examples/instancing.py

```diff
@@ -60,17 +59,23 @@ def __init__(self, width, height, title):
             void main() {
                 out_color = v_color;
             }
-            """
+            """,
         )
 
         self.instances = 1_000
         # Create triangle
-        vertices = array("f", [
-            # x, y
-            0.0, 0.8,
-            -0.8, -0.8,
-            0.8, -0.8,
-        ])
+        vertices = array(
+            "f",
+            [
+                # x, y
+                0.0,
+                0.8,
+                -0.8,
+                -0.8,
+                0.8,
+                -0.8,
+            ],
+        )
```

Comment:
I bit weird considering the comment is still there. When specifying geometry it can be nice to have have a primitive per line.